### PR TITLE
chore(release): 4.0.0

### PR DESCRIPTION
**Pull request by sfoster1**
Releaase 4.0.0 removes APIv1 compatibility and provides a significantly
better, faster, and more accurate calibration system.

### CODE REVIEW

File: api/src/opentrons/CHANGELOG.md [hunk 1]
```diff
@@ -3,235 +3,42 @@
...
+* **robot-server,api:** reset cal during dc process ([#6942](https://github.com/Opentrons/opentrons/issues/6942)) ([8d142d4](https://github.com/Opentrons/opentrons/commit/8d142d4))
+* **api:** recache position before plunger home ([#6881](https://github.com/Opentrons/opentrons/issues/6881)) ([603dfff](https://github.com/Opentrons/opentrons/commit/603dfff))
+* **api,robot-server:** reload robot calibration on deck calibration exit ([#6815](https://github.com/Opentrons/opentrons/issues/6815)) ([254af66](https://github.com/Opentrons/opentrons/commit/254af66))
+* **app:** Display if DC was migrated ([#6812](https://github.com/Opentrons/opentrons/issues/6812)) ([d37c0c0](https://github.com/Opentrons/opentrons/commit/d37c0c0))
+* **api:** Do not perform a transfer if a zero volume is passed in ([#6620](https://github.com/Opentrons/opentrons/issues/6620)) ([05e66b4](https://github.com/Opentrons/opentrons/commit/05e66b4))
+* **api:** Do not perform a transfer if a zero volume is passed in ([#6620](https://github.com/Opentrons/opentrons/issues/6620)) ([05e66b4](https://github.com/Opentrons/opentrons/commit/05e66b4))
```
Comment (b-cooper) [inline api/src/opentrons/CHANGELOG.md]: it looks like this is a redundant entry with the line above?

**Review by amitlissack — APPROVED** [review]

**Review by b-cooper — APPROVED** [review]
All looks good except one suspicious ditto

**Review by shlokamin — APPROVED** [review]
🚢

File: api/src/opentrons/CHANGELOG.md (hunk 1 above)
Comment (sfoster1) [reply api/src/opentrons/CHANGELOG.md]: Yeah because 3.21.2 happened during the 4.0 process they're interleaved

**Review by mcous — APPROVED** [review]
LGTM

**Review by sanni-t — APPROVED** [review]

**Pull request merged**